src/app.js

```diff
@@ -106,7 +106,7 @@ function init(env = process.env) {
       env.ALLOWED_IP_RANGES.split(" ").forEach((ipRange) => {
         allowedIps.push(ipRange.split("-"));
       });
-    } else if (env.NODE_ENV !== "testing") {
```

Comment:
Not sure why this didn't break previous builds since it was causing tests to fail on this PR...